### PR TITLE
fix(DB/Creature_loot): Increase drop rate of Ritual Salve

### PR DESCRIPTION
**Pull request by Maelthyrr**
## Changes Proposed:
-  Update drop rate to 72 % (based with test  on retail. Look below)

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/11875

## SOURCE:

Drop rate confirm also with https://wotlkdb.com/?item=6634

## Tests Performed:

- None

## How to Test the Changes:

1.Create a Tauren Shaman
2. Start Call of Earth questline from [Seer Ravenfeather](https://classic.wowhead.com/npc=5888/seer-ravenfeather)
3. See you are having difficulties gathering 2 Salves after more than 5 Shamans killed.

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Review by IntelligentQuantum — APPROVED** [review]

**Comment by Undeadhunter** [comment]
Tested OK  high drop rate but not 100% for sure on the Bristleback shamans

**Comment by Maelthyrr** [comment]
> Tested OK high drop rate but not 100% for sure on the Bristleback shamans

It's intend. Sources say 80%.

**Comment by Undeadhunter** [comment]
I badly worded that then, 
I didn't do a complete statistical analysis but out of the 25 kills I had 4 with no drop.

**Comment by acidmanifesto** [comment]
Be advise your link for reference states the site is from pserver data

**Comment by Maelthyrr** [comment]
I didn't know for woltkdb. But stay the same thing, videos show that the drop is near 100% but not 100%. And it's definitely not 14% as it is the case in the DB and WoWHead. What do we do ?

**Comment by acidmanifesto** [comment]
> I didn't know for woltkdb. But stay the same thing, videos show that the drop is near 100% but not 100%. And it's definitely not 14% as it is the case in the DB and WoWHead. What do we do ? 4/25 it's make 86%.

I will leave that for your discretion. This is where we will have to rely on what you feel is right. Im not home to check or match up with udb which has been a differ debate in itself as it doesnt cover conditions if any. Wowhead is diluted with their method of datamining thru addon use. But if you feel your approach from the video and other references would put it more towards a possible blizz like rate then go for it.

**Comment by Gultask** [comment]
See https://github.com/azerothcore/azerothcore-wotlk/pull/9379 as they had changed the drop rate for Ritual Salve. Now, I don't really know for sure but I think there was a mistake, as the table contained in the comments says 80% and in the sql it changed to 15%
Can't really be sure since I'm not on my PC right now, but I think 80% is the correct chance.

**Comment by neifion-00000000** [comment]
Currently working on this drop rate, how high confidence do we want to be on the rate?

ATM I am 99% confident that its drop rate is between 65 and 87%, and 90% confident it is between 69 and 83% but I am working on getting those numbers better.

**Comment by Maelthyrr** [comment]
> Currently working on this drop rate, how high confidence do we want to be on the rate?
> 
> (Updated)
> ATM I've mined enough data to know its 65-79% with 98% confidence  67-77 with 90% confidence
> 
> I'm currently 2/5ths of the amount I want to gather (500 samples) at which point I'd have a solid recommendation.

Thanks you si much 😊

File: data/sql/updates/pending_db_world/rev_1653810747543320600.sql [hunk 1]
```diff
@@ -0,0 +1,2 @@
+-- Adjust drop rate of Ritual salve to 72% (based by a sniffer with 325 kills)
+UPDATE  `creature_loot_template` SET `Chance` = 72 WHERE `Entry` = 2953 AND `Item` = 6634;
```
Comment (Nefertumm) [inline data/sql/updates/pending_db_world/rev_1653810747543320600.sql:2]:
```suggestion
UPDATE `creature_loot_template` SET `Chance` = 72 WHERE `Entry` = 2953 AND `Item` = 6634;
```

**Review by Nefertumm — CHANGES_REQUESTED** [review]

**Review by Nefertumm — APPROVED** [review]

**Pull request merged**